pabi_procurement/models/purchase_requisition.py

```diff
@@ -169,6 +169,15 @@ class PurchaseRequisition(models.Model):
         states={'draft': [('readonly', False)]},
         default=False,
     )
+    exclusive = fields.Selection(
```

Comment:
Is this field from standard Odoo, and you just want to have default?
If so, only what is overwritten is needed. I.e.,  `default='exclusive',`